cap-primitives/src/fs/readlink_one.rs

```diff
@@ -28,7 +29,7 @@ pub(crate) fn readlink_one(
     );
 
     if *symlink_count == MAX_SYMLINK_EXPANSIONS {
```

Comment:
I like the idea of making `readlink_one` platform-independent, but I have one concern: do you know if Windows also has a concept of a `MAX_SYMLINK_EXPANSIONS` in the way that Unix-type platforms do? We'd ideally match Windows path lookup behavior here, complete with symlink recursion detection.

Reply:
Good catch! Windows indeed does have a limit on reparse points and that limit currently is set at 63. I've now made `MAX_SYMLINK_EXPANSIONS` constant host-dependent, which effectively means we can still keep `readlink_one` host-independent, and we pull in from host-specific modules only the error and the constant.